Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -2066,3 +2076,17 @@ extension StatusTableViewController: ServicesViewModelDelegate {
     }
 
 }
+
+extension StatusTableViewController: GlucoseUnitPublisher {
+    func addGlucoseUnitObserver(_ observer: GlucoseUnitObserver, queue: DispatchQueue = .main) {
+        glucoseUnitObservers.insert(observer, queue: queue)
+    }
```

Comment:
Should this also include this chunk of code from `DeviceDataManager`'s implementation?
```
        if let glucoseUnit = glucoseStore.preferredUnit {
            observer.glucoseUnitDidChange(to: glucoseUnit)
        }
```

Reply:
Plus the same question about dispatching on the `queue`.

Reply:
I'll add it since I agree the pattern should be maintained. However, in the `StatusTVC` the preferred unit is provided to the view upon instantiation, just before it is added as an observer.